packages/theme/README.md

```diff
@@ -9,9 +9,17 @@ The Enact CLI itself can install this via NPM or git URI.
 enact template install @enact/template-theme
 ```
 
-## Usage
+## Creating a New Theme 
 
 Once installed, you can specify the `theme` template in the creation command:
 ```
-enact create -t theme MyTheme
+enact create -t theme uranium
 ```
+This will create a new copy of the starter theme in the `uranium` directory.  All references to the original theme's name, LESS variables, etc. will be updated to the appropriate values for `uranium`.
+
+The starter theme comes with one skin, `my-skin`, and this can be updated on creation as well:

```

Comment:
This may want to be reworded. The default skin when no `--skin <name>` becomes `default-skin`

Reply:
Ah, got it, thanks!